### PR TITLE
css-transitions -Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**

### CODE REVIEW

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes requested here, Lamberto! Please make sure to include a screenshot of the completed exercise

File: css-transitions/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  width: 100%;
+}
+
+.custom-button:hover {
+  transition: all 1s;
+  background-color: red;
+}
+
+.card:hover {
+  transition: all 0.5s;
```
[the remark this answers is not in the document]
Comment (JenBowman1) [reply css-transitions/styles.css:53]: hi Lamberto, as Shawn mentioned, please don't apply transition properties to pseudo selectors!

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
just one small change!

**Comment by Lambpato** [comment]
https://user-images.githubusercontent.com/34325124/217725205-13948a8f-0e19-413a-8b0b-677ce0e6e417.mov

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**